Objects/dictobject.c

```diff
@@ -7163,6 +7163,17 @@ PyObject_VisitManagedDict(PyObject *obj, visitproc visit, void *arg)
     return 0;
 }
 
+static void
+clear_inline_values(PyDictValues *values)
+{
+    if (values->valid) {
+        values->valid = 0;
```

Comment:
Does this need to be `FT_ATOMIC_STORE_UINT8(values->valid, 0)` for the call from `set_dict_inline_values`?